docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md

```diff
@@ -0,0 +1,22 @@
+In terminal, navigate to your project, run `amplify add auth`, and choose the following options (the last steps are specific to Facebook here but are similar for other providers):
+
+```terminal
+? Do you want to use the default authentication and security configuration? 
+    `Default configuration with Social Provider (Federation)`
+? How do you want users to be able to sign in? 
+    `Username`
+? Do you want to configure advanced settings? 
+    `No, I am done.`
+? What domain name prefix you want us to create for you? 
+    `(default)`
+? Enter your redirect signin URI: 
+    `myapp://`
```

Comment:
The Android docs say this should be `myapp://callback/`, and the redirect signout URI below should be ``myapp://signout/`.  Although, the iOS docs seem to make what you have here for Flutter.   Are you sure this is correct?  Does it matter?

Reply:
In iOS, the user is navigated back to wherever they launched the OAuth flow. We basically mimic them - on the Android side by navigating back to the  Android Activity that is created as part of our Flutter plugin.